lib/lookup.json

```diff
@@ -283,7 +283,7 @@
     "maintainers": "broofa"
   },
   "serialport": {
-    "flaky": ["ppc", "win32"],
+    "flaky": ["ppc", "win32", "ubuntu"],
```

Comment:
@ack.
P.S. I've opened a PR for `winston`: https://github.com/winstonjs/winston/pull/1028